docs/blockchain-users/FAQ.md

```diff
@@ -33,3 +37,60 @@ The [Discourse forum](https://quilibrium.discourse.group/) is intended to be the
 term. 
 It was a decision early in the project to embrace as much decentralized tools as possible, and if not present, that their absence serves as a forcing function to make them exist. 
 Discord especially, as the project started from a discord alternative. 
+
+### How fast is Quilibrium?
```

Comment:
I think until there is a production release, this section should be conservative and just say something like "Quilibrium was designed to support ultra-scale applications like Discord fully onchain" instead of specific numbers as they can change until the production release.